src/include/duckdb/common/arrow/arrow_options.hpp

```diff
@@ -13,6 +13,13 @@ namespace duckdb {
 enum ArrowOffsetSize { REGULAR, LARGE };
 
 struct ArrowOptions {
-	ArrowOffsetSize offset_size = ArrowOffsetSize::LARGE;
+	explicit ArrowOptions(ArrowOffsetSize offset_size_p) : offset_size(offset_size_p) {
+	}
+	ArrowOptions(ArrowOffsetSize offset_size_p, string timezone_p) : offset_size(offset_size_p), timezone(timezone_p) {
+	}
+	ArrowOptions() {
+	}
+	ArrowOffsetSize offset_size = ArrowOffsetSize::REGULAR;
+	string timezone = "UTC";
```

Comment:
To preemptively avoid issues like <https://github.com/duckdb/duckdb/pull/7258> can we not use `timezone` as a variable name and instead use `time_zone` ?